Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Logging/FtpLogger.cs

```diff
@@ -216,4 +216,4 @@ protected virtual void Dispose(bool disposing)
             _disposed = true;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Logging/FtpLogger.cs#L137
This method can be `static`.

https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Logging/FtpLogger.cs#L145
This method can be `static`.